.gitignore

```diff
@@ -68,3 +67,36 @@ iOS/nativelibs/arm64
 iOS/nativelibs/i386
 iOS/nativelibs/x86_64
 Solutions/Engine.App/test.ini
+
+libsrc/*.tar*
+libsrc/*.zip
+
+ac2game.dat
+audio.vox
+speech.vox
```

Comment:
libsrc/ files are downloaded as part of building native libraries for OSX, iOS and Android. 

The game files are sometimes in the project for testing (as they get added to app bundles)